src/Server/Services/Analytics/AnalyticsService.lua

```diff
@@ -728,29 +998,29 @@ function AnalyticsService:LogError(data: ErrorEvent): { [any]: any }
 		elseif data.severity ~= nil and not GameAnalytics.EGAErrorSeverity[data.severity] then
 			return reject("severity is invalid")
 		end
-		
-		local errorSeverity: string = data.severity or GameAnalytics.EGAErrorSeverity.Error
-		
-		GameAnalytics:addErrorEvent(data.userId, {
+
+		local errorSeverity: string = data.severity or GameAnalytics.EGAErrorSeverity.error
+
+		self:_wrapper("addErrorEvent", data.userId, {
 			message = data.message,
-			severity = GameAnalytics.EGAErrorSeverity[errorSeverity]
+			severity = GameAnalytics.EGAErrorSeverity[errorSeverity],
 		})
-		
-		return resolve()
+
+		resolve()
```

Comment:
```suggestion
		return resolve()
```
The linter/type checker complains if there's a return type defined for a function, but the promise is resolved/rejected without returning the function (For some reason)? It's an invalid error since we know we're returning a promise, but the code checker flags it for some reason.